gradle.properties

```diff
@@ -15,18 +15,15 @@ org.gradle.parallel.threads=8
 # This option should only be used with decoupled projects. More details, visit
```

Comment:
Because this is being published as open source, you should also define `licenseName`, `licenseUrl`, and `scmUrl` (it's used when publishing to Bintray). See here: https://github.com/rakutentech/android-remote-config/blob/master/gradle.properties#L4

Reply:
Thanks for sharing. It has been added now.